dask/bytes/pyarrow.py

```diff
@@ -0,0 +1,160 @@
+# to the license found at https://docs.python.org/3/license.html, which is also
+# included below:
+#
+#                PSF LICENSE AGREEMENT FOR PYTHON 3.6.4
+#                ======================================
+#
+# 1. This LICENSE AGREEMENT is between the Python Software Foundation ("PSF"),
+#    and the Individual or Organization ("Licensee") accessing and otherwise
+#    using Python 3.6.4 software in source or binary form and its associated
+#    documentation.
+#
+# 2. Subject to the terms and conditions of this License Agreement, PSF hereby
+#    grants Licensee a nonexclusive, royalty-free, world-wide license to
+#    reproduce, analyze, test, perform and/or display publicly, prepare
+#    derivative works, distribute, and otherwise use Python 3.6.4 alone or in
+#    any derivative version, provided, however, that PSF's License Agreement
+#    and PSF's notice of copyright, i.e., "Copyright c 2001-2016 Python
+#    Software Foundation; All Rights Reserved" are retained in Python 3.6.4
+#    alone or in any derivative version prepared by Licensee.
+#
+# 3. In the event Licensee prepares a derivative work that is based on or
+#    incorporates Python 3.6.4 or any part thereof, and wants to make the
+#    derivative work available to others as provided herein, then Licensee
+#    hereby agrees to include in any such work a brief summary of the changes
+#    made to Python 3.6.4.
+#
+# 4. PSF is making Python 3.6.4 available to Licensee on an "AS IS" basis.  PSF
+#    MAKES NO REPRESENTATIONS OR WARRANTIES, EXPRESS OR IMPLIED.  BY WAY OF
+#    EXAMPLE, BUT NOT LIMITATION, PSF MAKES NO AND DISCLAIMS ANY REPRESENTATION
+#    OR WARRANTY OF MERCHANTABILITY OR FITNESS FOR ANY PARTICULAR PURPOSE OR
+#    THAT THE USE OF PYTHON 3.6.4 WILL NOT INFRINGE ANY THIRD PARTY RIGHTS.
+#
+# 5. PSF SHALL NOT BE LIABLE TO LICENSEE OR ANY OTHER USERS OF PYTHON 3.6.4 FOR
+#    ANY INCIDENTAL, SPECIAL, OR CONSEQUENTIAL DAMAGES OR LOSS AS A RESULT OF
+#    MODIFYING, DISTRIBUTING, OR OTHERWISE USING PYTHON 3.6.4, OR ANY
+#    DERIVATIVE THEREOF, EVEN IF ADVISED OF THE POSSIBILITY THEREOF.
+#
+# 6. This License Agreement will automatically terminate upon a material breach
+#    of its terms and conditions.
+#
+# 7. Nothing in this License Agreement shall be deemed to create any
+#    relationship of agency, partnership, or joint venture between PSF and
+#    Licensee.  This License Agreement does not grant permission to use PSF
+#    trademarks or trade name in a trademark sense to endorse or promote
+#    products or services of Licensee, or any third party.
+#
+# 8. By copying, installing or otherwise using Python 3.6.4, Licensee agrees to
+#    be bound by the terms and conditions of this License Agreement.
+#
+# These functions are under copyright by the Python Software Foundation
+#
+#    Copyright 2001-2018 Python Software Foundation; All Rights Reserved
+
+
+def _glob(fs, pathname):
```

Comment:
Do you think it's useful to export glob for other file-systems? 
(I know there are issues on the glob implementation elsewhere)